packages/ts/src/components/graph/config.ts

```diff
@@ -143,8 +143,12 @@ export interface GraphConfigInterface<N extends GraphInputNode, L extends GraphI
   nodeIconSize?: NumericAccessor<N>;
   /** Node label accessor function or constant value. Default: `node => node.label` */
   nodeLabel?: StringAccessor<N>;
+  /** Defines whether to trim the node labels or not. Default: `true` */
+  nodeLabelTrim?: BooleanAccessor<N>;
```

Comment:
@reb-dev We should, but I wanted to do it later along with overall refactoring of labels in the library. However, it can be a small change, I'll check

Reply:
@reb-dev  I've added additional properties to configure trimming and extended the example accordingly

<img width="1161" alt="image" src="https://user-images.githubusercontent.com/755708/216708190-5852addb-0123-49ae-9b31-b44c5ea0b7cb.png">

<img width="895" alt="image" src="https://user-images.githubusercontent.com/755708/216708103-115423ad-049f-42a4-b756-598d9f0096e4.png">

Reply:
Great!